packages/driver/src/cy/stability.ts

```diff
@@ -1,58 +1,48 @@
 import Promise from 'bluebird'
 
-const tryFn = (fn) => {
-  // promisify this function
-  return Promise.try(fn)
-}
-
-export default {
-  create: (Cypress, state) => {
-    const isStable = (stable = true, event) => {
-      if (state('isStable') === stable) {
-        return
-      }
-
-      const whenStable = state('whenStable')
-
-      // if we are going back to stable and we have
-      // a whenStable callback
-      if (stable && whenStable) {
-        // invoke it
-        whenStable()
-      }
-
-      state('isStable', stable)
-
-      // we notify the outside world because this is what the runner uses to
-      // show the 'loading spinner' during an app page loading transition event
-      return Cypress.action('cy:stability:changed', stable, event)
+// eslint-disable-next-line @cypress/dev/arrow-body-multiline-braces
```

Comment:
If you had maintained the code structure here, would you have needed to disable this rule?
```
export const create = (Cypress, state) => {
  const isStabe = () => {}
  const whenStable = () => {}
  return {
    isStable,
    whenStable,
  }
}
```

Reply:
I structured them in this way because I want to emphasize all functions are exported. When we use `return`, we cannot be sure if it's exported or not until we check if it's included in `return`. I didn't follow this convention there are internal functions or functions that are shared between other functions.